entrypoint.sh

```diff
@@ -40,20 +42,15 @@ DATADIR="/home/erigon-gnosis/.local/share"
 exec erigon --datadir=${DATADIR} \
     --chain=gnosis \
     --http.addr=0.0.0.0 \
-    --http.port=8545 \
     --http.vhosts=* \
     --http.corsdomain=* \
     --ws \
     --private.api.addr=0.0.0.0:9090 \
     --metrics \
     --metrics.addr=0.0.0.0 \
     --metrics.port=6060 \
-    --pprof \
-    --pprof.addr=0.0.0.0 \
-    --pprof.port=6061 \
     --port=${P2P_PORT} \
     --authrpc.jwtsecret=${JWT_PATH} \
     --authrpc.addr=0.0.0.0 \
     --authrpc.vhosts=* \
-    --torrent.port=43084 \
```

Comment:
We should probably keep this with a configurable environment variable. Torrent sync is quite important on Erigon and opening the ports by default would be great.